app/views/console/storage/index.phtml

```diff
@@ -22,7 +22,7 @@
                 data-scope="sdk"
                 data-name="project-buckets">
 
-                <div data-ls-if="(!{{project-buckets.total}})" class="box dashboard margin-bottom">
+                <div data-ls-if="0 == {{project-buckets.total}} || undefined == {{project-buckets.total}}" class="box dashboard margin-bottom">
```

Comment:
Both are falsy values, so shouldnt `data-ls-if="{{project-buckets.total}}"` do the trick?

Reply:
Silly me. The existing code was correct . 

It was the old `X-Appwrite-Response-Format` header that was causing the error